docs/rules/prefer-numeric-literals.md

```diff
@@ -0,0 +1,41 @@
+# disallow `parseInt` in favor of binary, octal, and hexadecimal literals (prefer-numeric-literals)
+
+Binary and octal literals were introduced in ES6. Prior to this, `parseInt` was used to turn binary and octal strings into integers. This rule encourages use of binary, octal, and hexadecimal literals instead of `parseInt`.
```

Comment:
Can you use `parseInt()` with parens throughout? We try to stick to that convention in the docs.